### PR TITLE
rewrite trento community check regarding 'running corosync rings' 

### PR DESCRIPTION
**Pull request by angelabriel**
fixing jsc#CFSA-1076, check 1.1.9.runtime/32CFC6

### CODE REVIEW

**Review by arbulu89 — APPROVED** [review]
Perfect!

File: priv/catalog/32CFC6.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+values:
+  - name: expected_totem_interfaces
+    default: 2
+    conditions:
+      - value: 1
+        when: env.provider == "azure" || env.provider == "gcp"
+
+expectations:
+  - name: expectations_totem_interfaces
+    expect: facts.totem_interfaces.len() == values.expected_totem_interfaces
```
Comment (nelsonkopliku) [inline priv/catalog/32CFC6.yaml:49]: Does the `at least` particle in the description mean we need `>=`? 
Also in relation to discussions had here about the check  **DA114A** https://github.com/trento-project/wanda/pull/141#issuecomment-1361642249

**Review by nelsonkopliku — APPROVED** [review]
Super! Just a tiny doubt, but we're good.

File: priv/catalog/32CFC6.yaml (hunk 1 above)
Comment (angelabriel) [reply priv/catalog/32CFC6.yaml:49]: oh, oh, you are so right. Will fix it asap. Many thanks for the attention

**Pull request merged**